content/blog/cpu-profiling-2/index.md

```diff
@@ -0,0 +1,188 @@
+---
+path: '/cpu-profiling'
+title: 'Part 2: A Simple eBPF Continuous Cross-Language (Go, Rust, C/C++) Profiler'
```

Comment:
What do you think about "Part 2: A Simple Continuous Cross-Language (Go, Rust, C/C++) Profiler written in eBPF"